game-guide/trading.md

```diff
@@ -1,54 +1,297 @@
+Experience Gain = (Cargo Holds / 13) &middot; Distance Factor
 
-On a strictly trading basis, if you are making more than 16 non trading moves a day the IST is ahead of the PSF for experience earned. Of course the PSF is much more defendable, but the IST can trade anywhere in the universe quite easily, so is much less predictable.
+Moving goods with a greater distance factor is a huge part of gaining experience.  The person who moves the most goods along the longest route will usually end up with the most.
 
-One final note: In order to gain Federal protection at the end of your trading session, you CANNOT have any illegal goods on board. So make sure that you have all of them sold off before heading to Fed, and make sure that you have sold off your weapons to get your attack rating down. (See Alignment for more detail on attack ratings and Federal Protection.)
+Experience is also tricky because of the bartering process.  There is an ideal price that must be guessed quite accurately in order to get full experience.  But the port will offer you a price that, if your relations are not at least 1000, will not be optimal.  Guessing the correct value is incredibly difficult due to the number of variables used in the process.  Most people opt simply to work their relations up as fast as possible.
 
-~~The next four subsections will give a few more detailed tidbits of information to help you on your quest to be a top trader.~~
+## Relations
 
-Read XDemonX's Guide to Smart Trading~~
\ No newline at end of file
+Every player has two types of relation numbers for each race.  There are personal relations which are what the members of that race think of you personally.  And there are political relations which represent what races collectively think of each other.  Your own actions directly influence personal relations.  Political relations are adjusted through council actions.
+
+Racial ruling councils are made up with the top 5 experience players from that race.  Each of the 5 members are able to vote to adjust relations with another race each day.  Each of those votes adds or subtracts 15 political relations from one other race.  
+
+The highest experience council member one with 150+ alignment is made president.  The president has the power to call or veto votes.  Votes can trigger war, which sets political relations with another race to -400, or peace, which sets political relations with another race to 300.  While the president holds more power in this regard, the majority on the council ultimately has more authority.
+
+The war and peace votes don't matter as much as you may think.  Any race that gets to -300 is considered to be at war.  Any race that gets to 300 is considered to be at peace.  The declaration doesn't have to happen via a vote called by the president.
```

Comment:
```suggestion
The war and peace votes don't matter as much as you may think.  Any race that gets to -300 is considered to be at war.  Any race that gets to 300 is considered to be at peace.  The declaration doesn't have to happen via a vote called by the president. Instead, it can happen slowly over time by the daily relations votes by council members. 
```